src/index.d.ts

```diff
@@ -1529,6 +1629,23 @@ declare namespace admin.auth {
      */
     getUserByPhoneNumber(phoneNumber: string): Promise<admin.auth.UserRecord>;
 
+    /**
+     * Gets the user data corresponding to the specified identifiers.
+     *
+     * There are no ordering guarantees; in particular, the nth entry in the result list is not
+     * guaranteed to correspond to the nth entry in the input parameters list.
+     *
+     * Only a maximum of 100 identifiers may be supplied. If more than 100 identifiers are supplied,
```

Comment:
I think this method rejects when array.length > 100. Let's verify and update the comment.

Reply:
i.e. throw vs failed promise? It should throw. (There's a test to ensure it here: test/unit/auth/auth.spec.ts:1142.)

Reply:
Ah, I see now. Thanks. The terminology used in the test case descriptions could need some work though.